docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -100,16 +107,33 @@ and reject the whole document.
 
 |=======================================================================
 
+
+[[indexing-approach]]
+[float]
+==== Indexing approach
+GeoShape types are indexed by decomposing the shape into a triangular mesh and
```

Comment:
maybe we could mention that indexing performance mostly depends on the number of edges of the input (multi)polygon

Reply:
:+1: